kolibri/utils/cli.py

```diff
@@ -14,10 +15,44 @@
 
 import kolibri  # noqa
 from kolibri import dist as kolibri_dist  # noqa
+from kolibri import dist_cext as kolibri_dist_cext
+
 sys.path = sys.path + [
     os.path.realpath(os.path.dirname(kolibri_dist.__file__))
 ]
 
+
+"""
+Get the directory of dist_cext.
+"""
+# Directory with the c modules
+dirname = os.path.dirname(kolibri_dist_cext.__file__)
+# Python version of current platform
+python_version = 'cp' + str(sys.version_info.major) + str(sys.version_info.minor)
+dirname = os.path.join(dirname, python_version)
+
+# Current platform
+platform = util.get_platform()
+# For Linux system, there could be abi tags 'm' and 'mu'
+if 'linux' in platform and int(python_version[2:]) < 33:
+    dirname = os.path.join(dirname, 'linux')
+    # encode with ucs2
+    if sys.maxunicode == 65535:
+        dirname = os.path.join(dirname, python_version+'m')
+    # encode with ucs4
+    else:
+        dirname = os.path.join(dirname, python_version+'mu')
+elif 'macosx' in platform:
+    platform = 'macosx'
```

Comment:
Sorry do you mean the logic here looks like a duplicate of the logic in install_cext.py? So the logic in install_cext.py is to parse the whl files in pypi repositories and distribute them into different local directories depending on different platforms. The logic in this file is to get the platform of the running system and to find whether the corresponding c extension exists in local. In this case, I don't feel like we can combine these two. But I might misunderstand what you said, could you elaborate more on the duplicate logic part? Thank you!